samples/DirectX/D3D12/D3D12HelloTexture.cpp

```diff
@@ -0,0 +1,546 @@
+//*********************************************************
```

Comment:
The C++ files should not be checked in, they provide no use in the actual codebase and should just be referenced instead (as the samples have already been doing).

Reply:
removed it and the reference to it.